lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -5875,108 +6123,104 @@ ADContext::declareExternalAssociatedFunction(
   return assocFn;
 }
 
-static SILFunction* createJVP(
+static SILFunction *createEmptyVJP(
```

Comment:
Renamed this to more closely resemble naming and logic of `VJPEmitter` which creates an empty VJP and then uses an emitter to generate the body